handler/index.js

```diff
@@ -0,0 +1,69 @@
+import cors from './middlewares/cors';
+import globalErrorHandler from './middlewares/globals/error-handler';
+import requestLoggerMiddleware from './middlewares/globals/request-logger';
+
+const setResponseHeaders = (headers = [], response) =>
+  Object.keys(headers).forEach((name) => {
+    const value = headers[name];
+    response.setHeader(name, value);
+  });
```

Comment:
[Não impeditivo, só pra conhecimento :sweat_smile:] 
Nesses casos o  `Object.entries` funciona bem, ficaria algo como: 
```suggestion
const setResponseHeaders = (headers = [], response) => 
  Object.entries(headers).forEach(([name, value]) => 
    response.setHeader(name, value));
```

Reply:
Pô, que massa, não lembrava desse carinha aí, ficou bem melhor. Vou commitar :heart_eyes: